install_config/topics/using_gcp_load_balancer.adoc

```diff
@@ -0,0 +1,119 @@
+balancer in GCP and creates the proper firewall rules.
+
+The following is an example `LoadBalancer` service:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: hello-openshift
+  name: hello-openshift-external
+spec:
+  externalTrafficPolicy: Cluster
+  ports:
+  - name: port-1
+    nodePort: 30714
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  - name: port-2
+    nodePort: 30122
+    port: 8888
+    protocol: TCP
+    targetPort: 8888
+  selector:
+    app: hello-openshift
+    deploymentconfig: hello-openshift
+  sessionAffinity: None
+  type: LoadBalancer
+----
+
+.Procedure
+
+. Create a new application:
++
+[source,bash]
+----
+$ oc new-app openshift/hello-openshift
+----
+
+. Expose the load balancer service:
++
+[source,bash]
+----
+$ oc expose dc hello-openshift --name='hello-openshift-external' --type='LoadBalancer'
+----
+
+. To verify that the service has been created:
++
+[source,bash]
+----
+$ oc get svc
+NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                         AGE
+hello-openshift            ClusterIP      172.30.62.10     <none>          8080/TCP,8888/TCP               20m
+hello-openshift-external   LoadBalancer   172.30.147.214   35.230.97.224   8080:31521/TCP,8888:30843/TCP   19m
+----
++
+The `LoadBalancer` type and External IP fields indicate that the service is
+using GCP load balancers to expose the application.
+
+{product-title} creates the required objects in the GCP infrastructure such as:
+
+* Firewall rules:
++
+[source,bash]
+----
+$ gcloud compute firewall-rules list | grep k8s
+k8s-4612931a3a47c204-node-http-hc        my-net  INGRESS    1000      tcp:10256
+k8s-fw-a1a8afaa7762811e88c5242010af0000  my-net  INGRESS    1000      tcp:8080,tcp:8888
+----
++
+[NOTE]
+====
+The above firewall rules are applied to instances tagged with
+`<openshift_gcp_prefix>ocp`. For example, if the value of the
+`openshift_gcp_prefix` parameter is set to `mycluster`, you must tag the nodes
+with `myclusterocp`. See
+link:https://cloud.google.com/vpc/docs/add-remove-network-tags[Adding and
+Removing Network Tags] for more information on how to add network tags to
+Compute Engine VM instances.
+====
+
+* Health checks:
++
+[source,bash]
+----
+$ gcloud compute http-health-checks list | grep k8s
+k8s-4612931a3a47c204-node        10256  /healthz
+----
+
+* A load balancer:
++
+[source,bash]
+----
+$ gcloud compute target-pools list | grep k8s
+a1a8afaa7762811e88c5242010af0000  us-west1  NONE                      k8s-4612931a3a47c204-node
+$ gcloud compute forwarding-rules list | grep a1a8afaa7762811e88c5242010af0000
+a1a8afaa7762811e88c5242010af0000  us-west1  35.230.97.224  TCP          us-west1/targetPools/a1a8afaa7762811e88c5242010af0000
+----
+
+To verify that the load balancer is properly configured, run the following from an external host:
+
+[source,bash]
+----
+$ curl 35.230.97.224:8080
```

Comment:
What IP address is this? Please annotate.

Reply:
The exposed service IP (see line 67)